src/pybind/mgr/dashboard/base.html

```diff
@@ -413,6 +413,13 @@
             <li class="ceph-none-found" rv-hide="filesystems | length">None found</li>
           </ul>
         </li>
+
+        <li class="{%if path_info.startswith('/rgw')%}active{%endif%}">          
+            <a href="{{ url_prefix }}/rgw">                                                        
+ <i class="fa fa-files-o" aria-hidden="true"></i>
```

Comment:
There is no "right" icon for RGW (as far as I know), but what about cloud-download (http://fontawesome.io/icon/cloud-download/) instead of a file-related icon?